README.md

```diff
@@ -107,6 +108,8 @@ alt="platformatic"
 * `noStyle` - *`boolean`* - if set, no style rules will be added. Especially useful when combined with [Prettier](https://prettier.io/), [dprint](https://dprint.dev/) or similar
 * `semi` - *`boolean`* - if set, enforce rather than forbid semicolons (same as `semistandard` did)
 * `ts` - *`boolean`* - if set, TypeScript syntax will be supported and `*.ts` (including `*.d.ts`) will be checked. To add additional file patterns to the TypeScript checks, use `filesTs`
+* `react` - *`'recommended' | 'all'`* - selected rule set from `eslint-plugin-react`, see [JSX support](#jsx-support) for details
```

Comment:
As I've already explained, neostandard goal is to propose a controlled configuration to prescribe a consistent linting and formatting behavior:

+ Relying on external rules set we do not control is not compliant
+ Extract from the default rules set (jsx-runtime) the ones that match with the existing ones for JS/TS

And you've not answered: what would be the impact of enabling unconditionally JSX support on JS/TS behavior?  

Tests are also missing.

Reply:
I'm not sure how to read your message... Let me make 1 thing crystal clear: I'm just trying to help. I have no intention to be a "Contributor" or anything. I was told to create a PR, so I did. If my help is not wanted, I have plenty of other work to do. Just tell me... no hard feelings, no offense taken.

You have explained that, and I tried to explain I'm not an author of any `neostandard` rule. My intention was not to define more rules or change any of the existing ones. To enable JSX support I had to install external plugin, which does define a set of other rules. Without their rules, React application with neostandard installed would not work (because it would miss import of React, for example). That's just 2 of their rules (`react/react-in-jsx-scope` and `react/jsx-uses-react`). And there are other, really handy rules, like `react/jsx-key`. Again, I do not want to decide which rules to adapt and which not. Or create any new rules. But without their rules, it will be useless as you won't get any React support. Just neostandard rules working in JSX/TSX files. People will still have to find their way how to enable another plugin and set it up. That I managed to do for JSX files without touching your code, but not for TSX.
> what would be the impact of enabling unconditionally JSX support on JS/TS behavior?
I do not know, how to answer your question, because I do not understand what are you asking about. If I manage to enable neostandard rules in JSX/TSX files, by default, without adding any new rules from external plugin, I think it won't help at all, because users will still have to figure out how to enable other plugins, and will end up having their own eslint configuration, not simplified neostandard with 1 line of code in eslint.config.js.
> Tests are also missing.
True. I'm not going to spend more time on tests until it is clear what I have to do to have JSX support in `neostandard`

Reply:
> You have explained that, and I tried to explain I'm not an author of any `neostandard` rule. My intention was not to define more rules or change any of the existing ones. 

Extract the minimal configuration to get JSX/TSX linting and formatting to work in neostandard from the shareable configurations (eslint plugin, ...) without introducing any rules set tunable that might change the neostandard behavior. Just like TS support does not permit to choose any of the existing shareable configurations in TS ESlint plugin. If one wants to use TS ESLint shareable configurations with neostandard, one merge them in his config: https://github.com/SAP/e-mobility-charging-stations-simulator/blob/main/eslint.config.js

1:1 mapping between existing JS and TS rules and the ones matching for JSX/TSX support if needed: it's not rules addition or proposition, it's properly implementing JSX/TSX support by ensuring any existing behavior for JS/TS will be replicated for JSX/TSX. As it was done for TS support: JS behavior replicated in TS.

Shareable or custom configs support in neostanstard API is not the goal of that PR.

It's the third time I rephrase it, and I do not know how to make it more understandable. 

> > what would be the impact of enabling unconditionally JSX support on JS/TS behavior?
> 
> I do not know, how to answer your question, because I do not understand what are you asking about. 

If reactify() is enable unconditionally, is there any behavioral change on JS/TS?